README.md

```diff
@@ -1,3 +1,17 @@
 # Pulsar Edit
 
 This is the website repo for Pulsar Edit. It contains our documenation and other information users might be looking for. Visit the [Website](https://pulsar-edit.github.io).
+
+
+## Guide
+
+### Tools Needed:
+- Toolbox: We recommend using this as a development tool. It helps with containerization of development.
+  - This can be setup in most distributions of linux using your package manager or with brew. Brew can also be used on Mac but if that's not ideal we recommend getting directly from the [official website](https://containertoolbx.org/).
```

Comment:
Maybe once we have more details we could go ahead and make each OS it's own bullet point, like so:

```markdown
- Toolbox: We recommend using this as a development tool. It helps with containerization of the development process.
  - Linux: ...
  - Mac: ...
  - Windows: ...
```

Reply:
I mean, there's also Vagrant like @autumnblazey mentioned in discord. I've just no clue in writing images to make sure our dev environments are synced up. It's why I went the route of toolbox, since it makes it simple without me having to touch things with writing an image. Totally open to changing that, and would end up with Toolbox under Linux as an option or having Vagrant with some pre-made images based on the person's OS/packaging flavor IMO.

Reply:
I was more meaning this as then to detail instructions of the tool for the different OS's like so. For example:

```markdown 
- Toolbox: ...
  - Linux: Can be set up in most distributions using your package manager or with brew.
  - Mac: Can be setup with Brew, or otherwise setup using the [Official Website]()
  - Windows: ... idk Get rekt or something
```

Reply:
Yea just gonna close this conversation. This is something we need to discuss and change to other tooling so that people's stuff doesn't break when helping us. Preferably vagrant or something like that (if we can find a universal gui for setting this up…).